src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -382,6 +402,28 @@ private static string GetBucket(AugmentedTimelineIssue augIssue)
             return await azureServer.ListBuilds(project, cancellationToken, minDateTime, limit);
         }
 
+        const string refsHeadsPrefix = "refs/heads/";
+        public static string NormalizeBranchName(string branch)
```

Comment:
I think you can use https://github.com/dotnet/arcade-services/blob/2dcf38b83b249927c33369e0272b17245de95dd2/src/Microsoft.DotNet.Darc/src/DarcLib/IGitRepoExtension.cs#L43-L50 for this.

Reply:
I mostly just wasn't excited about pulling all of DarcLib into the TelemetryApplication, but if that doesn't seem like a big deal, I'll change it.

Reply:
Move it out of darc lib. :-)

Reply:
Just want to mention that while I would be ok moving this function (or maybe everything in IGitRepoExtension.cs) to a shared utility dll, there are 10 references to this function and 3 to GetDecodedContent, so I'll be adding a number of files that currently reference these functions. @mmitche how do you feel about this being moved to utilities (which doesn't currently exist).

Reply:
Ok I moved the function.